### PR TITLE
Fixed interface error

### PR DESCRIPTION
**Pull request by angelnikolov**

### CODE REVIEW

**Review by tonytlwu — APPROVED** [review]

File: js/interface.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ Fliplet().then(() => {
     audioUrlInput.val('https://open.spotify.com/track/2YarjDYjBJuH63dUIh9OWv?si=PlS4yiAWT9afuC2UyVKCeA').trigger('change');
   });
 
-  if (widgetInstanceData.embedlyData.url) {
+  if (widgetInstanceData.embedlyData && widgetInstanceData.embedlyData.url) {
```
Comment (tonytlwu) [inline js/interface.js:116]:
```suggestion
  if (widgetInstanceData && _.hasIn(widgetInstanceData, 'embedlyData.url')) {
```

Comment (tonytlwu) [reply js/interface.js:116]: This is how I usually test for long list of property chains

Comment (angelnikolov) [reply js/interface.js:116]: Is there a reason to use that for a simple condition? I would use `hasIn` for a deeply nested object property path.

Comment (tonytlwu) [reply js/interface.js:116]: No preference. That's why I approved it. Just telling you my usual way of doing it :)

**Pull request merged**